contrib/pinot-fmpp-maven-plugin/pom.xml

```diff
@@ -66,6 +66,12 @@
       <groupId>net.sourceforge.fmpp</groupId>
       <artifactId>fmpp</artifactId>
       <version>${fmpp.version}</version>
+      <exclusions>
```

Comment:
Do we need to exclude freemarker here after explicitly specifying its version?

Reply:
yes, otherwise, it reports version conflict